web/src/deb/resources/etc/georchestra/geonetwork/config/config-overrides-georchestra.xml

```diff
@@ -4,6 +4,7 @@
     <!-- earlier imported from config-security/config-security.xml -->
     <import file="${env:georchestra.datadir}/geonetwork/config/config-security-georchestra.xml" />
     <import file="${env:georchestra.datadir}/geonetwork/config/config-db-georchestra.xml" />
+    <import file="${env:georchestra.datadir}/geonetwork/config/config-datadir-georchestra.xml" />
     <import file="${env:georchestra.datadir}/geonetwork/config/config-logging-georchestra.xml" />
```

Comment:
this is kind of risky for debian packages, because you won't have the rights to write into /etc/georchestra/geonetwork for tomcat user, basically